stdlib/3/typing.pyi

```diff
@@ -220,11 +220,11 @@ class Container(Protocol[_T_co]):
 
 if sys.version_info >= (3, 6):
     @runtime
-    class Collection(Sized, Iterable[_T_co], Container[_T_co], Protocol[_T_co]): ...
+    class Collection(Sized, Iterable[_T_co], Container[_T_co], Protocol[_T_co], metaclass=ABCMeta): ...
```

Comment:
Hm, I think this should be not needed. Having an explicit `Protocol` base should be enough. Protocols can't be instantiated at runtime anyway.

Reply:
Done, changed it in mypy here: https://github.com/python/mypy/pull/5223/commits/e505c0d6195270bdf8bd351bceca3c7a05d6c5e2.